styleguide/_themes/derek/scss/components/_footer.scss

```diff
@@ -1,3 +1,117 @@
 .footer{
-  
-}
\ No newline at end of file
+  color:#fff;
+  ul{
+    list-style: none;
+    padding: 0px;
+    margin:0px;
+    li{
+      padding: 0px;
+      margin:5px 0px;
+      a{
+        color:#aaa;
+        font-size: 1.2rem;
+      }
+      span.menu-title{
+        font-size: 1.3rem;
+      }
+    }
+  }
+}
+/*  Mobile   */
+@media only screen and (min-width:50px) and (max-width: 767px)  {
+  .fluid-footer-container{
+    padding: 0px 0px;
+  }
+  .footer{
+    width: 100%;
+    padding: 0px 0px;
+    ul{
+      li{
+        padding: 0px;
+        margin:0px 0px;
+        text-align: center;
+        display: block;
+        width: 100%;
+        span.menu-title{
+          border-bottom:1px solid #2E3238;
+          background-color:#23262b;
+          padding:12px;
+          width:100%;
+          display: block;
+          font-size: 1.3rem;
+          &:hover{
+            cursor: pointer;
+            cursor: hand;
+          }
+        }
+        ul{
+          li{
+            ul{
+              li{
+                padding:0px;
+                border-bottom: 1px solid #3b4047;
+                a{
```

Comment:
I'd tend to agree with this.  

@runrog I can force drupal to output depth classes on the <li>'s.  This would probably be sufficient in zoolander as well.  

Instead of that `ul li ul li ul li` nesting, you can just do `ul li.depth-3`.  That should help cleanup that css a bit.